spec/javascripts/components/toggle-input-class-on-focus-spec.js

```diff
@@ -14,20 +14,20 @@ describe('A toggle class module', function () {
     beforeEach(function () {
       element = $(
         '<div data-module="gem-toggle-input-class-on-focus">' +
-          '<input type="search" class="js-class-toggle"/>' +
+          '<input type="search" class="js-class-toggle" id="test"/>' +
         '</div>')
       $('body').append(element)
       toggle = new GOVUK.Modules.GemToggleInputClassOnFocus()
       toggle.start(element)
     })
 
     it('applies the focus style on focus and removes it on blur', function () {
-      var searchInput = element.find('.js-class-toggle')
-      expect(searchInput.is('.focus')).toBe(false)
-      searchInput.triggerHandler('focus')
-      expect(searchInput.is('.focus')).toBe(true)
-      searchInput.triggerHandler('blur')
-      expect(searchInput.is('.focus')).toBe(false)
+      var searchInput = document.getElementById('test')
```

Comment:
non-blocker: why the aditional `id` and not `document.querySelector('.js-class-toggle')` to match the same selector with the script?

Reply:
Good question. No real reason, have changed, thanks. I think it maybe initially failed and I switched it to this to be certain the vanilla JS was playing correctly with the events being triggered in the test, and forgot to switch it back.